generator/generator_tool/generator_tool.cpp

```diff
@@ -205,6 +210,10 @@ int GeneratorToolMain(int argc, char ** argv)
 
   google::ParseCommandLineFlags(&argc, &argv, true);
 
+  auto countTrheads = std::thread::hardware_concurrency();
```

Comment:
s/Trheads/Threads/

also, threadsCount and numThreads are more customary

Reply:
++